nbgrader/exchange/ngshare/collect.py

```diff
@@ -34,23 +32,15 @@ def _get_submission(self, course_id, assignment_id, student_id):
         dictionary containing the "path" relative to the assignment root and the
         "content" as an ASCII representation of the base64 encoded bytes.
         """
-        url = '{}{}/submission/{}/{}/{}'.format(self.ngshare_url, self.prefix,
-                                                course_id, assignment_id,
-                                                student_id)
-        params = {'user': self.username}
-
-        try:
-            response = requests.get(url, params=params)
-        except:
-            self.log.error('An error occurred downloading a submission.')
-            return None
 
-        if response.status_code != requests.codes.ok or not response.json()['success']:
+        response = self.ngshare_api_get("/submission/{}/{}/{}".format(
```

Comment:
is there a reason why we don't pass in the username to the get request anymore?

Reply:
that is only used for vngshare, not proper ngshare